v3/integrations/nrfasthttp/instrumentation.go

```diff
@@ -50,7 +50,7 @@ func WrapHandle(app *newrelic.Application, pattern string, handler fasthttp.Requ
 	if app == nil {
 		return pattern, handler
 	}
-
+	newrelic.GetSecurityAgentInterface().SendEvent("API_END_POINTS", pattern, "*", internal.HandlerName(handler))
```

Comment:
This should be enclosed in a check for `IsSecurityAgentPresent` to avoid needlessly calling `internal.HandlerName` and the `GetSecurityAgentInterface` and `SendEvent` calls.